src/routes/[...slug]/+error.svelte

```diff
@@ -0,0 +1,131 @@
+<script>
+	import { page } from '$app/stores';
+
+	function goBack() {
+		window.history.back();
+	}	
+
+	function navigateTo(url) {
+		window.location.href = url;
+	}
+
+	let pathParts = $page.url.pathname.split('/').filter(Boolean);
+
+	function isCurrentPage(index) {
+		return index === pathParts.length - 1;
+	}
+</script>
+
+<body>
+	<nav>
+		<ul>
+			<li>
+				<button class="breadcrumbs {pathParts.length === 0 ? 'current' : ''}" on:click={() => navigateTo('/')}>home</button>
+			</li>
+			{#each pathParts as part, index (index)}
+				<li>
+					<button
+						class="breadcrumbs {isCurrentPage(index) ? 'current' : ''}"
+						on:click={() => navigateTo(`/${pathParts.slice(0, index + 1).join('/')}`)}
+						>{part}</button
+					>
```

Comment:
Ook voor de breadcrumbs geen buttons gebruiken, maar links. Het kunnen door- en deeplinken naar url's is een van de belangrijkste principes van het web!

Reply:
Breadcrumbs in een component zetten en op alle pagina's includen?

Reply:
Ik heb hier ook buttons gebruikt omdat dit ook in de error pagina is en hier geen CSS wordt meegegeven om een of andere reden (zie wat ik hier 2 comments boven zei) misschien weet je een oplossing die mijn css wel meegeeft?

Breadcrumbs op alle pagina's is een goed idee ik zal dat zeker doen!

Reply:
Ik kijk woensdag even met je mee!